### DISCUSSION TITLE
Location of qucsconv? qucsconv does not come with Qucs-s

### DISCUSSION THREAD

**Discussion by asdgdg355235** [ra3xdh/qucs_s#606] [Q&A] [answered]
I've installed qucs-s from both the aur and from compiling it myself, in both cases qucsconv is not available. Even if I go in the converter folder the only program I can compile is vcd2qucsdat, which I assume is only for converting vcd to qucsdata. I can not find any information on how to install qucsconv manually. I've been trying to convert a asc file, is this no longer supported?

**Comment by ra3xdh** [comment, marked as answer]
Qucsconv is a part of qucsator https://github.com/qucs/qucsator . This repository is maintained by another team and I cannot directly include it in the Qucs-S distribution. Qucsator development is stalled now. You have to compile qucsator to obtian qucsconv. See also #370

**Comment by tomhajjar** [comment]
What is your goal with Qucsconv? If the goal is to convert Spice models, Qucsconv supplied with Qucs only works for simple Spice models.

"asc file" generally means LTspice schematics...
